tails_files/install.sh

```diff
@@ -20,15 +40,44 @@ apt-get update
 apt-get install -y build-essential
 gcc -o $SCRIPT_BIN securedrop_init.c
 
-# copy init scripts
-cp securedrop_init.py $SCRIPT_PY
-cp .xsessionrc $INSTALL_DIR
-
+# copy icon, launchers and scripts
 cp securedrop_icon.png $INSTALL_DIR
+cp document.desktop $INSTALL_DIR
+cp source.desktop $INSTALL_DIR
+cp .xsessionrc $INSTALL_DIR
+cp securedrop_init.py $SCRIPT_PY
 
-# prepare torrc_additions
-cp torrc_additions $ADDITIONS
-gedit $ADDITIONS > /dev/null 2>&1
+if $ADMIN; then
+  DOCUMENT=`cat $ANSIBLE/app-document-aths | cut -d ' ' -f 2`
+  SOURCE=`cat $ANSIBLE/app-source-ths`
+  APPSSH=`cat $ANSIBLE/app-ssh-aths | cut -d ' ' -f 2`
+  MONSSH=`cat $ANSIBLE/mon-ssh-aths | cut -d ' ' -f 2`
+  cat $ANSIBLE/app-ssh-aths $ANSIBLE/mon-ssh-aths $ANSIBLE/app-document-aths >> $ADDITIONS
+  if [[ -d "$HOMEDIR/.ssh" && ! -f "$HOMEDIR/.ssh/config" ]]; then
+    # create SSH host aliases and install them
+    SSHUSER=$(zenity --entry --title="Admin SSH user" --window-icon=$INSTALL_DIR/securedrop_icon.png --text="Enter your username on the App and Monitor server:")
+    cat > $INSTALL_DIR/ssh_config <<EOL
+Host app
+  Hostname $APPSSH
+  User $SSHUSER
+Host mon
+  Hostname $MONSSH
+  User $SSHUSER
+EOL
+    chown amnesia:amnesia $INSTALL_DIR/ssh_config
+    chmod 600 $INSTALL_DIR/ssh_config
+    cp -p $INSTALL_DIR/ssh_config $HOMEDIR/.ssh/config
+    SSH_ALIASES=true
+  fi
+  # set ansible to auto-install
+  if ! grep -q 'ansible' "$TAILSCFG/live-additional-software.conf"; then
+    echo "ansible" >> $TAILSCFG/live-additional-software.conf
+  fi
+else
+# prepare torrc_additions (journalist)
```

Comment:
Please put a better commnet. "The journalist workstation does not have the `*-aths` files automatically created by the Ansible playbook, so the configuration file must be updated manually. Open a graphical text editor so the journalist can easily paste the `HidServAuth` info they will receive from the Admin" or something like that.